tests/test_app/config/AltSeeds/NumbersAltSeed.php

```diff
@@ -18,7 +18,6 @@ public function run()
     {
         $data = [
             [
-                'id' => '1',
```

Comment:
We can. Tests pass without problems with the change.
How it will behave is tied to the schema of your database : it will be no problem if you have this column defined as autoincrement (which is the case here).
[All examples of seed in phinx doc](http://docs.phinx.org/en/latest/seeding.html#using-the-table-object) doesn't include a value for the PK fields (well, I suppose there is a PK since there's no schema provided)